tests/test_perturbation.py

```diff
@@ -86,10 +86,9 @@ def test_perturbation_matrix3d():
     dx, dy = dx.ravel(), dy.ravel()
 
     # ntime x npixels
-    flux = (
-        np.random.normal(0, 0.01, size=(10, 169))
-        + 2 * dx[None, :] ** 2 * 3 * dy[None, :]
-    )
+    flux = np.random.normal(0, 0.01, size=(10, 169)) + dx[None, :] * dy[None, :]
```

Comment:
Such a nit to pick but you can do 
```suggestion
    flux = np.random.normal(1, 0.01, size=(10, 169)) + dx[None, :] * dy[None, :]
```

Reply:
And then get rid of line below

Reply:
great!